tests/ts/JavaScriptRequiredStringTest.js

```diff
@@ -0,0 +1,33 @@
+import assert from 'assert'
+import * as flatbuffers from 'flatbuffers';
+// import * as flatbuffers from '~/src/mediasoup/node_modules/flatbuffers/js'
+import { Foo } from './required-strings/foo.js';
+
+
+var builder = new flatbuffers.Builder();
+
+function main() {
+  testMissingRequiredStringA();
```

Comment:
Not sure what the ts/js style for testing, I would just do something simple like
```
  const undefined_string = builder.createString(undefined);
  const defined_string = builder.createString('cat');

	assert.throws(() => Foo.createFoo(
		builder, undefined_string, defined_string
	));

	assert.throws(() => Foo.createFoo(
		builder, defined_string, undefined_string
	));
```

Reply:
done